dz2.1/ДЗ. TypeScript. Анотація типів та примітивні типи.ts

```diff
@@ -0,0 +1,116 @@
+class School {
+  directions: Direction[];
+
+  addDirection(direction: Direction) {
+    this.directions.push(direction);
+  }
+}
+
+class Direction {
+  levels: Level[];
+
+  get name() {
+    return this._name;
+  }
+
+  constructor(name: string) {
+    this._name = name;
+  }
+
+  addLevel(level: Level) {
```

Comment:
Please, do not forget to explicitly add type annotation for the return function value.